RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1865,5 +1865,22 @@ public bool VolunteerHasAlreadyJobForThisRequestWithThisStatus(int jobId, int vo
                 return false;
             }
         }
+
+        public List<int> GetOverdueRepeatJobs()
+        {
+            List<int> response = new List<int>();
+            byte jobStatusNew = (byte)JobStatuses.New;
+            byte jobStatusOpen = (byte)JobStatuses.Open;
+
+            DateTime now = DateTime.UtcNow;
+
+            response = _context.Job
+                .Include(x => x.NewRequest)
+                .Where(x => x.NewRequest.Repeat == true && x.DueDate< now && (x.JobStatusId == jobStatusNew || x.JobStatusId == jobStatusOpen) )
```

Comment:
What DueDate do we put in for the first repeat in the series?  Just wondering if it's DateTime.Now - in which case that job would always get cancelled the first time this runs.

Best solution could be to additionally check for the DueDateType in this code, and not cancel anything that's DueDateType.ASAP

Reply:
exclude DueDateType.ASAP